docs/swagger/spec.yml

```diff
@@ -2233,6 +2199,38 @@ definitions:
           enabled:
             type: boolean
 
+  SingleBandOptions:
+    type: object
+    properties:
+      band:
+        type: number
+        format: integer
+      dataType:
+        type: string
+      blendMode:
+        type: string
+      colorScheme:
+        type: object
+      legendOrientation:
+        type: string
+
+  ColorCorrection:
+    type: object
+    properties:
+      isSingleBand:
+        type: boolean
+      singleBandOptions:
+        $ref: '#/definitions/SingleBandOptions'
+      rbgBandOptions:
```

Comment:
It's actually also probably superfluous to have an isSingleBand option if we make it oneOf